plugin/build.gradle.kts

```diff
@@ -126,6 +130,9 @@ dependencies {
     compileOnly("org.openrewrite:rewrite-protobuf")
     compileOnly("org.openrewrite:rewrite-xml")
     compileOnly("org.openrewrite:rewrite-yaml")
+    compileOnly("org.openrewrite.recipe:rewrite-all:$latest") {
+        isTransitive = false
```

Comment:
Is disabling transitives actually necessary for some reason?

Reply:
Not necessary. Imagine the idea was to just bring in `OmniParser` without all the other recipe modules